testground/benchmark/benchmark/sendtx.py

```diff
@@ -78,15 +80,51 @@ def generate_load(num_accounts, num_txs, global_seq, **kwargs):
                 print("test task failed", e)
 
 
-def prepare_txs(global_seq, num_accounts, num_txs):
+def async_write_tx(account, tx, file_path):
+    raw_tx = account.sign_transaction(tx).rawTransaction
+    with open(file_path, "w") as file:
+        file.write(ujson.dumps([raw_tx.hex()]))
+
+
+def get_txs_foler(outdir: Path, group, global_seq):
+    folder = outdir / TMP_TXS_FOLDER / group / str(global_seq)
+    folder.mkdir(parents=True, exist_ok=True)
+    return folder
+
+
+def prepare_txs(
+    outdir: Path, group, global_seq, num_accounts, num_txs, max_workers=8000
+):
+    folder = get_txs_foler(outdir, group, global_seq)
     accounts = [gen_account(global_seq, i + 1) for i in range(num_accounts)]
-    txs = []
-    for i in range(num_txs):
-        for acct in accounts:
-            txs.append(acct.sign_transaction(test_tx(i)).rawTransaction.hex())
-            if len(txs) % 1000 == 0:
-                print("prepared", len(txs), "txs")
+    with ThreadPoolExecutor(max_workers=max_workers) as executor:
+        futs = []
+        for acc_i in range(num_accounts):
+            for tx_i in range(num_txs):
+                file_path = folder / f"tx_{acc_i}_{tx_i}.json"
+                if file_path.exists():
+                    continue
+                futs.append(
+                    executor.submit(
+                        async_write_tx, accounts[acc_i], test_tx(tx_i), file_path
+                    )
+                )
+
+        for fut in futs:
+            try:
+                fut.result()
+            except Exception as e:
+                print("sign_transaction failed", e)
+    print(f"prepared {len(futs)} txs for {group} {global_seq}")
 
+
+def read_txs(outdir: Path, group, global_seq):
+    txs = []
+    folder = get_txs_foler(outdir, group, global_seq)
+    for file in folder.glob("*.json"):
```

Comment:
why multiple files for single node?

Reply:
I find it hard to reuse for single txs.json instead of global_seq/tx_{acc_i}_{tx_i}.json

Reply:
just `{global_seq}.json`?